ssh-audit.py

```diff
@@ -3713,14 +3720,55 @@ def audit(aconf: AuditConf, sshv: Optional[int] = None, print_target: bool = Fal
 
     return program_retval
 
+def algorithm_lookup(alg_names):       
+    if not alg_names:    
+        out.fail("[exception] Missing argument: alg_names")
```

Comment:
The program handles all final argument checking & validation in `AuditConf.from_cmdline()`.  It would be nice to move this checking there.